<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1174</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1174</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_clean">
    <property name="title">
     <string>清洗</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_analysis">
    <property name="title">
     <string>分析</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_window">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="menu_window_file"/>
    <addaction name="menu_window_stack"/>
    <addaction name="menu_window_table"/>
    <addaction name="menu_window_graph"/>
    <addaction name="menu_window_config"/>
    <addaction name="menu_window_history"/>
    <addaction name="separator"/>
    <addaction name="menu_window_savestore"/>
    <addaction name="menu_window_restore"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_clean"/>
   <addaction name="menu_analysis"/>
   <addaction name="menu_window"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_datafiles">
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>文件</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="datafiles_list"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_config">
   <property name="windowTitle">
    <string>参数配置</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>解析数据文件参数</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_20">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>csv文件分隔符</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="config_datafiles_csv_seperator">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>词典</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_11">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>停用词典</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="config_stop_words_dict">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_stop_words_dict">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_12">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>合并词典</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="config_combine_words_dict">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_combine_words_dict">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_14">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>受控词典</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="config_controlled_words_dict">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_controlled_words_dict">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save_config">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_history">
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>操作日志</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_stack">
   <property name="windowTitle">
    <string>栈</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget_stack"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_table">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>表格</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="TableKit" name="clean_datatable"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_graph">
   <property name="windowTitle">
    <string>图像</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6"/>
  </widget>
  <widget class="QToolBar" name="clean_toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="menu_clean_save">
   <property name="icon">
    <iconset>
     <normaloff>icons/xiazai.png</normaloff>icons/xiazai.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="menu_clean_metadata">
   <property name="icon">
    <iconset>
     <normaloff>icons/biaoge.png</normaloff>icons/biaoge.png</iconset>
   </property>
   <property name="text">
    <string>元数据</string>
   </property>
  </action>
  <action name="menu_clean_rename">
   <property name="icon">
    <iconset>
     <normaloff>icons/gengduo.png</normaloff>icons/gengduo.png</iconset>
   </property>
   <property name="text">
    <string>重命名</string>
   </property>
  </action>
  <action name="menu_copy_column">
   <property name="icon">
    <iconset>
     <normaloff>icons/lieziduan.png</normaloff>icons/lieziduan.png</iconset>
   </property>
   <property name="text">
    <string>复制列</string>
   </property>
  </action>
  <action name="menu_split_column">
   <property name="icon">
    <iconset>
     <normaloff>icons/zhongmingming.png</normaloff>icons/zhongmingming.png</iconset>
   </property>
   <property name="text">
    <string>拆分列</string>
   </property>
  </action>
  <action name="menu_replace_column">
   <property name="icon">
    <iconset>
     <normaloff>icons/bumenpaixu.png</normaloff>icons/bumenpaixu.png</iconset>
   </property>
   <property name="text">
    <string>替换值</string>
   </property>
  </action>
  <action name="menu_combine_synonym">
   <property name="icon">
    <iconset>
     <normaloff>icons/shujufenxi.png</normaloff>icons/shujufenxi.png</iconset>
   </property>
   <property name="text">
    <string>合并词</string>
   </property>
  </action>
  <action name="menu_stop_words">
   <property name="icon">
    <iconset>
     <normaloff>icons/wenben.png</normaloff>icons/wenben.png</iconset>
   </property>
   <property name="text">
    <string>停用词</string>
   </property>
  </action>
  <action name="menu_compare_columns">
   <property name="icon">
    <iconset>
     <normaloff>icons/lieziduan.png</normaloff>icons/lieziduan.png</iconset>
   </property>
   <property name="text">
    <string>对比列</string>
   </property>
  </action>
  <action name="menu_modify_value">
   <property name="icon">
    <iconset>
     <normaloff>icons/riqi.png</normaloff>icons/riqi.png</iconset>
   </property>
   <property name="text">
    <string>修改值</string>
   </property>
  </action>
  <action name="menu_row_distinct">
   <property name="icon">
    <iconset>
     <normaloff>icons/shuzi.png</normaloff>icons/shuzi.png</iconset>
   </property>
   <property name="text">
    <string>行去重</string>
   </property>
  </action>
  <action name="menu_row_similarity">
   <property name="icon">
    <iconset>
     <normaloff>icons/biaochaxun.png</normaloff>icons/biaochaxun.png</iconset>
   </property>
   <property name="text">
    <string>相似度</string>
   </property>
  </action>
  <action name="menu_row_delete">
   <property name="icon">
    <iconset>
     <normaloff>icons/yichu.png</normaloff>icons/yichu.png</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
  </action>
  <action name="menu_column_delete">
   <property name="icon">
    <iconset>
     <normaloff>icons/yichu2.png</normaloff>icons/yichu2.png</iconset>
   </property>
   <property name="text">
    <string>删除列</string>
   </property>
  </action>
  <action name="menu_split_words">
   <property name="icon">
    <iconset>
     <normaloff>icons/xianshi.png</normaloff>icons/xianshi.png</iconset>
   </property>
   <property name="text">
    <string>切分词</string>
   </property>
  </action>
  <action name="menu_column_delete_2">
   <property name="text">
    <string>补全值</string>
   </property>
  </action>
  <action name="menu_clean_undo">
   <property name="icon">
    <iconset>
     <normaloff>icons/chexiao.png</normaloff>icons/chexiao.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="menu_clean_redo">
   <property name="icon">
    <iconset>
     <normaloff>icons/huifu.png</normaloff>icons/huifu.png</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="menu_group_stat">
   <property name="text">
    <string>分组统计</string>
   </property>
  </action>
  <action name="menu_library_freq_stat">
   <property name="text">
    <string>频次统计</string>
   </property>
  </action>
  <action name="menu_library_cocon_stat">
   <property name="text">
    <string>共现统计</string>
   </property>
  </action>
  <action name="menu_cocon_stat">
   <property name="icon">
    <iconset>
     <normaloff>icons/yingyongzhongxin.png</normaloff>icons/yingyongzhongxin.png</iconset>
   </property>
   <property name="text">
    <string>共现分析</string>
   </property>
  </action>
  <action name="menu_count_stat">
   <property name="icon">
    <iconset>
     <normaloff>icons/yingyongchangjing.png</normaloff>icons/yingyongchangjing.png</iconset>
   </property>
   <property name="text">
    <string>词频统计</string>
   </property>
  </action>
  <action name="menu_clean_filter">
   <property name="text">
    <string>过滤行</string>
   </property>
  </action>
  <action name="menu_clean_makeup">
   <property name="text">
    <string>补全值</string>
   </property>
  </action>
  <action name="menu_vertical_concat">
   <property name="icon">
    <iconset>
     <normaloff>icons/vertical_concat.png</normaloff>icons/vertical_concat.png</iconset>
   </property>
   <property name="text">
    <string>数据合并</string>
   </property>
  </action>
  <action name="menu_clean_parse">
   <property name="icon">
    <iconset>
     <normaloff>icons/aislogo.png</normaloff>icons/aislogo.png</iconset>
   </property>
   <property name="text">
    <string>解析</string>
   </property>
  </action>
  <action name="menu_clean_graph_config">
   <property name="icon">
    <iconset>
     <normaloff>icons/app.png</normaloff>icons/app.png</iconset>
   </property>
   <property name="text">
    <string>图表配置</string>
   </property>
  </action>
  <action name="menu_clean_extract_feature">
   <property name="icon">
    <iconset>
     <normaloff>icons/zhenshikexin.png</normaloff>icons/zhenshikexin.png</iconset>
   </property>
   <property name="text">
    <string>特征提取</string>
   </property>
  </action>
  <action name="menu_window_savestore">
   <property name="text">
    <string>保存布局</string>
   </property>
  </action>
  <action name="menu_window_restore">
   <property name="text">
    <string>恢复布局</string>
   </property>
  </action>
  <action name="menu_window_file">
   <property name="text">
    <string>文件窗口</string>
   </property>
  </action>
  <action name="menu_window_stack">
   <property name="text">
    <string>栈窗口</string>
   </property>
  </action>
  <action name="menu_window_table">
   <property name="text">
    <string>表格窗口</string>
   </property>
  </action>
  <action name="menu_window_graph">
   <property name="text">
    <string>图像窗口</string>
   </property>
  </action>
  <action name="menu_window_config">
   <property name="text">
    <string>参数窗口</string>
   </property>
  </action>
  <action name="menu_window_history">
   <property name="text">
    <string>操作窗口</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableKit</class>
   <extends>QTableView</extends>
   <header>core.toolkit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
